Makefile

```diff
@@ -933,15 +925,15 @@ clean-release-git: ## Restores the git files usually modified during a release
 
 .PHONY: clean-generated-yaml
 clean-generated-yaml: ## Remove files generated by conversion-gen from the mentioned dirs. Example SRC_DIRS="./api/v1beta1"
-	(IFS=','; for i in $(SRC_DIRS); do find $$i -type f -name '*.yaml' -exec rm -f {} \;; done)
+	(IFS=','; for i in $(SRC_DIRS); do find $$i -type f -name '*.yaml' -exec rm -f {} || true \;; done)
```

Comment:
I think in the past this found a few errors because it fails if the SRC_DIRS are misconfigured. Are we sure we want to ignore this going forward?

Reply:
I have dropped it, it is not core to the PR.
However,  If after running make generate-manifests, clean happens, but then generation fails, then you cannot run make generate-manifests again.

Reply:
Yup, that's the downside I'm aware :) Just prefer this one over:
* we configure the wrong paths in SRC_DIRS
* target doesn't fail
* we have another issue in generation
* we never notice our files get outdated